packages/web-twig/src/Resources/components/Accordion/README.md

```diff
@@ -156,43 +156,43 @@ This is Twig implementation of the [Accordion] component.
-| ------------- | -------- | ----------- | -------- | ------------------------ |
-| `id`          | `string` | `null`      | no/yes   | Optional, depends on ... |
-| `class`       | `string` | `null`      | no       | Additional class name    |
-| `elementType` | `string` | `'section'` | no       | Custom element type      |
+| Prop name     | Type     | Default     | Required | Description                                                                                           |
+| ------------- | -------- | ----------- | -------- | ----------------------------------------------------------------------------------------------------- |
+| `id`          | `string` | `null`      | no/yes   | Optional, It depends on whether the "Always open" option is used, if so, the id field is not required |
+| `class`       | `string` | `null`      | no       | Additional class name                                                                                 |
+| `elementType` | `string` | `'section'` | no       | Custom element type                                                                                   |
 
 ### AccordionItem
 
 | Prop name     | Type     | Default     | Required | Description           |
 | ------------- | -------- | ----------- | -------- | --------------------- |
-| `id`          | `string` | `null`      | yes      | Accordion Item ID     |
+| `id`          | `string` | `null`      | yes      | AccordionItem ID      |
 | `class`       | `string` | `null`      | no       | Additional class name |
 | `elementType` | `string` | `'article'` | no       | Custom element type   |
 
 ### AccordionHeader
 
-| Prop name     | Type     | Default | Required | Description              |
-| ------------- | -------- | ------- | -------- | ------------------------ |
-| `id`          | `string` | `null`  | yes      | AccordionHeader ID       |
-| `class`       | `string` | `null`  | no       | Additional class name    |
-| `for`         | `string` | `null`  | yes      | Target element selector  |
-| `elementType` | `string` | `'h3'`  | no       | Custom element type      |
-| `slot`        | `HTML`   | ``      | no       | Side slot in the header  |
-| `isOpen`      | `string` | `false` | no       | Whether is item selected |
+| Prop name     | Type     | Default | Required | Description                  |
+| ------------- | -------- | ------- | -------- | ---------------------------- |
+| `id`          | `string` | `null`  | yes      | AccordionHeader ID           |
+| `class`       | `string` | `null`  | no       | Additional class name        |
+| `for`         | `string` | `null`  | yes      | AccordionContent ID          |
+| `elementType` | `string` | `'h3'`  | no       | Custom element type          |
+| `slot`        | `HTML`   | ``      | no       | Side slot in the header      |
+| `isOpen`      | `string` | `false` | no       | Whether the item is expanded |
```

Comment:
What about staying with the `open` status?

```suggestion
| `isOpen`      | `string` | `false` | no       | Whether the item is open |
```

Reply:
fixed